context/app/static/js/components/Detail/ProvTabs/utils.js

```diff
@@ -0,0 +1,7 @@
+export function checkDataTypesForValues(dataTypes, valuesToCheck) {
```

Comment:
Consider renaming, perhaps `hasDataTypes`? 

`check` is ambiguous, and what the return value should be gets muddled if we're looking for a good thing or a bad thing.